### DISCUSSION TITLE
Unknown drugs in prescriptions table: "Bag" and "SW"

### DISCUSSION THREAD

**Discussion by LV3ga** [MIT-LCP/mimic-code#1405] [MIMIC-IV] [answered]
We encountered these drugs in the prescriptions table within the hosp module. 

Both of these drugs have no gsn/ndc code. Does anyone know what these drugs are? Where could we find a reference?

**Comment by marymlucas** [comment, marked as answer]
Hi @LV3ga you can find the name of the drug for each row by joining to the Pharmacy table in hosp module on 'pharmacy_id'.

Some sample code to try on BigQuery that accomplishes this:
```
with bag_sw as 
  (select * from `physionet-data.mimic_hosp.prescriptions` where drug="Bag" or drug="SW")
    select a.hadm_id, a.pharmacy_id, a.drug, b.medication from bag_sw a left join  `physionet-data.mimic_hosp.pharmacy` b
             on a.pharmacy_id  = b.pharmacy_id;
```

You can also cross-check with emar table, but I had better success with pharmacy table.

**Reply by LV3ga** [reply]
Thank you so much!
